### PR TITLE
fix(docs): fixes and edits into docs for code snippets

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**
## Proposed Changes

_[describe the changes here...]_

## Linked Issues

_[if applicable, add links to issues resolved by this PR]_

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [x] Content update.
- [ ] Bug fix (non-breaking change that fixes an issue).
- [ ] New feature added (non-breaking change that adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).

## Checklist

_Put an `x` in the boxes that apply:_

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/docs/blob/main/CONTRIBUTING.md) guide
- [x] Checks and tests pass locally

## Further comments

_[if this is a relatively large or complex change, kick off a discussion by explaining why you chose the solution you did, what alternatives you considered, etc...]_

### CODE REVIEW

File: pages/examples/jupyter-agent.mdx [hunk 1]
```diff
@@ -21,249 +21,14 @@ Are you new to Agents? To understand the agent in this example, please read [com
 
 ## Notebook
 
-<GithubCodeSegment digest="964b533f69f19c4c4e407ce900471ab4">
+<GithubCodeSegment digest="">
```
Comment (devjsc) [inline pages/examples/jupyter-agent.mdx]: This needs updating to new path

**Review by devjsc — CHANGES_REQUESTED** [review]

File: pages/examples/jupyter-agent.mdx (hunk 1 above)
Comment (FelixNicolaeBucsa) [reply pages/examples/jupyter-agent.mdx]: Having a look

Comment (FelixNicolaeBucsa) [reply pages/examples/jupyter-agent.mdx]: Solved

File: .github/spelling/known_words_corpus.txt [hunk 2]
```diff
@@ -720,4 +720,49 @@ aws
 guidesmdx
 agentsicon
 ledgericon
-asiicon
\ No newline at end of file
+asiicon
+ganet
+googleapis
+toolresult
+xvfb
```
Comment (devjsc) [inline .github/spelling/known_words_corpus.txt:727]: This isn't a word 😅

**Review by devjsc — CHANGES_REQUESTED** [review]

File: .github/spelling/known_words_corpus.txt (hunk 2 above)
Comment (FelixNicolaeBucsa) [reply .github/spelling/known_words_corpus.txt:727]: That is in a code snippet

Comment (FelixNicolaeBucsa) [reply .github/spelling/known_words_corpus.txt:727]: In `pages/guides/quickstart-with/Anthropic/computer-use-demo.mdx`

Comment (devjsc) [reply .github/spelling/known_words_corpus.txt:727]: does the code spell checker fail for spelling in code blocks?

Comment (FelixNicolaeBucsa) [reply .github/spelling/known_words_corpus.txt:727]: Nope,. it checks both content and block as far as i remember correctly but provides separate views in terminal

**Pull request merged**